src/chat/Chat.js

```diff
@@ -33,11 +33,11 @@ export default class Chat extends PureComponent<Props> {
   props: Props;
 
   handleSend = () => {
-    setTimeout(() => {
```

Comment:
We need to do this in both way. i.e when new message is arrived and new message is sent (only if already scrolled to end.) Will be working on it next :)

For now I commented, as it is throwing error when there is switch in components (no message and messageList).

Reply:
Can you check somehow if listComponent is mounted?

Reply:
Tried a bit today. But didn't got any good work around. If we increase 300 to 1300, it is working fine, but will not be good solution.